/Users/vinitmehta/Desktop/IIITH/Sem5/SMAI/Assignments/smai-m24-assignments-Vinit2244/models/MLP/MLP.py:118: RuntimeWarning: invalid value encountered in multiply
  err = np.dot(err, self.weights[i].T) * self.activation_func_derivative(activation)
Model Training completed successfully in time 2.0929253101348877 s!
